easybuild/easyconfigs/m/mcqd/mcqd-1.0.0-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CmdCp'
+
+name = 'mcqd'
+version = '1.0.0'
+
+homepage = 'https://gitlab.com/janezkonc/mcqd'
+description = """MaxCliqueDyn is a fast exact algorithm for finding a maximum clique in an undirected graph."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
```

Comment:
@PetrKralCZ Let's use `GCC` for this one, because the compiler used (GCC or Intel compilers) may imply impact on performance. Note that this implies also dropping the `binutils` build dep and renaming the easyconfig

Reply:
changed